spec/sidekiq/prometheus/exporter_spec.rb

```diff
@@ -60,14 +65,29 @@
         instance_double(Sidekiq::Queue, name: 'additional', size: 0, latency: 1.00200001)
       ]
     end
+    let(:now) { Time.now }
+    let(:workers) do
+      [
+        ['worker1:1:0493e4117adb', '2oe', {'queue' => 'default', 'run_at' => now.to_i - 10, 'payload' => {}}],
+        ['worker1:1:0493e4117adb', '2si', {'queue' => 'default', 'run_at' => now.to_i - 20, 'payload' => {}}],
+        ['worker2:1:dbf573ecf819', '2hi', {'queue' => 'additional', 'run_at' => now.to_i - 30, 'payload' => {}}],
+        ['worker2:1:dbf573ecf819', '2s8', {'queue' => 'additional', 'run_at' => now.to_i - 40, 'payload' => {}}]
+      ]
+    end
 
     before do
       allow(Sidekiq::Stats).to receive(:new).and_return(stats)
       allow(Sidekiq::Queue).to receive(:all).and_return(queues)
+      allow(Sidekiq::Workers).to receive(:new).and_return(workers)
 
+      Timecop.freeze(now)
```

Comment:
Can I ask you to pull it to allow statements and keep `get` action separated by blank line?